core-bundle/src/Doctrine/Backup/Dumper.php

```diff
@@ -76,7 +80,7 @@ private function disableQueryBuffering(Connection $connection): void
     }
 
     /**
-     * @param AbstractSchemaManager<AbstractPlatform> $schemaManager
+     * @phpstan-param AbstractSchemaManager<AbstractPlatform> $schemaManager
```

Comment:
Why did you change this?

Reply:
I did not, the CS fixer did.

Reply:
There is no such rule in our fixer setup. 🤔 Anyways, I have reverted it in b846408d87dc5f782ca8b802dff149988fa59faa.